models/demos/t3000/llama3_70b/README.md

```diff
@@ -56,28 +56,33 @@ After setting up the repacked weights and tokenizer, you can run the demo using
 4. **Run the performance test:**
 
     The above demo does not achieve peak performance because we log outputs to the screen. The following perf test will print an accurate end-to-end throughput number.
-    For best performance numbers, we recommend building `tt-metal` with `CONFIG=Release` env var, and ensuring the host's CPU frequency governors are set to `performance`.
+    For best performance, ensure that tt-metal is built in release mode (default), and ensure the host's CPU frequency governors are set to `performance` -- instructions for setting the frequency governor vary by machine.
+    This performance test runs with sequence length 128 and batch size 32.
     ```bash
     pytest -svv models/demos/t3000/llama2_70b/tests/test_llama_perf_decode.py::test_Llama_perf_host[wormhole_b0-True-device_params0-gen128-llama3]
     ```
 
 ## Details
 
-- **Batch Size:** Supports batch size 16 and 32.
+Supported context lengths and batch sizes for the Llama3.1-70B demo are as follows:
+
+| Context Length | Max Batch Size |
+|----------------|------------|
+| 2k             | 32         |
+| 8k             | 16         |
+| 128k            | 1         |
+
 - **Input File:** Uses `./demo/data/multi_prompt.json`.
 - **Model Configuration:** Utilizes a pretrained model.
 - **Hardware Requirements:** Runs on an 8-chip T3000 machine using tensor parallelism. The host machine must have at least 512 GB of memory.
-- **Model Functionality:**
-    - The demo can run in `decode_only` mode in which we use decode mode to consume the context one token at a time, or `prefill_decode` mode in which we prefill the context and then decode.
-
 - **Demo arguments:**
-    - `context: [short_context, long_context]`: Select between short context (batch 32, sequence_length 2k) and long context (batch 16, sequence length 8k)
+    - `context: [short_context, long_context, 128k_context]`: Select between short context (batch 32, sequence_length 2k) and long context (batch 16, sequence length 8k) and full context (batch 1, sequence length 128k)
```

Comment:
Ideally, this should be an integer so the user can select any context length since it affects the performance and the max batch size. We can implement this later.